java/clients/src/test/resources/reports/filestatus/standard/splitFile.txt

```diff
@@ -2,15 +2,14 @@
 Files Status Report:
 --------------------------
 There are 2 leaf partitions and 1 non-leaf partitions
-There are 1 files with status of "Ready_to_be_garbage_collected"
-	(0 in leaf partitions, 1 in non-leaf partitions)
+There are 1 files with no references, which are ready to be garbage collected
 There are 4 files with status of "Active"
 	(3 in leaf partitions, 1 in non-leaf partitions)
 Number of files in leaf partitions: min = 1, max = 2, average = 1.5
 Number of files in non-leaf partitions: min = 1, max = 1, average = 1.0
-Ready_to_be_garbage_collected:
-FileInfo{filename='split.parquet', partitionId='A', numberOfRecords=2000, fileStatus=READY_FOR_GARBAGE_COLLECTION, jobId='null', lastStateStoreUpdateTime=1661178000001, countApproximate=false, onlyContainsDataForThisPartition=true}
-Active:
+Files with no references:
+[split.parquet]
```

Comment:
It would be good to report on which of these are really ready to be garbage collected, and when the others will be ready. We can split out a separate issue for that.

Reply:
We can rethink this later, it's not clear whether this would be valuable.